fission-web-server/library/Fission/Web/Server/Auth/DID.hs

```diff
@@ -30,8 +30,9 @@ handler ::
   -> m DID
 handler req =
   case Token.get req of
-    Right (Token.Bearer token@(Bearer.Token jwt rawContent)) ->
-      JWT.check rawContent jwt >>= \case
+    Right (Token.Bearer token@(Bearer.Token jwt rawContent)) -> do
+      serverDID <- getServerDID
+      JWT.check serverDID rawContent jwt >>= \case
```

Comment:
No longer has an implicit recipient in the UCAN, so we can do CLI<>browser linking. Needs to be passed in explicitly here for the server use case now ☝️